src/main/java/seedu/address/logic/parser/TagDeleteCommandParser.java

```diff
@@ -0,0 +1,32 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.logic.commands.TagDeleteCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.tag.Tag;
+
+
+/**
+ * Parses input arguments and creates a new FindCommand object
+ */
+public class TagDeleteCommandParser implements Parser<TagDeleteCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the TagDeleteCommand
+     * and returns an TagDeleteCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public TagDeleteCommand parse(String args) throws ParseException {
```

Comment:
What if there is space character in the tag? If the space happens to be before/after the tag, you might want to trim the argument before adding it to Tag.